homeassistant/components/device_tracker/owntracks.py

```diff
@@ -28,10 +28,18 @@
 
 LOCK = threading.Lock()
 
+CONF_MAX_GPS_ACCURACY = 'max_gps_accuracy'
+DEFAULT_MAX_GPS_ACCURACY = 500
+
 
 def setup_scanner(hass, config, see):
     """ Set up an OwnTracks tracker. """
 
+    if CONF_MAX_GPS_ACCURACY in config:
+        max_gps_accuracy = config[CONF_MAX_GPS_ACCURACY]
```

Comment:
No `if` statement needed if you use `get` on a `dict`. It will return `None` (or the optional second argument) if not found.
```python
max_gps_accuracy = config.get(CONF_MAX_GPS_ACCURACY)
```